File: CS50/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //declare variables
    int height;
    
    //prompt user for input between 1 and 7
    do 
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    
    
    //print out # pyramid
    for (int i = 0; i < height; i++)
    {
        for(int k = (height-1); k > i ; k--)
        {
            printf(" ");
        }
        
        for(int j = 0; j <= i; j++)
        {
            printf("#");   
        }
        
        printf("\n");
       
    }
}

File: CS50/pset6/mario/less/mario.py
from cs50 import get_int

#get user input between 1 and 8
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    num = i+1
    print((height-num) * " ", end="")
    print(num * "#", end="")
    print("")
    